### PR TITLE
Limit center freq

### PR DESCRIPTION
**Pull request by jthermiz**
Only calculate wavelet for frequencies 2.5 times less than sample rate

### CODE REVIEW

**Comment by JesseLivezey** [comment]
I'd advocate for a different logic here. In general, I think it's better to either do what the user asked or raise an error. Changing things behind the scenes (even if documented) would be somewhat surprising.

I'd suggest raising an error if the max cfs is above the nyqist (2x). Then, as a lab, we should use the 2.5x standard.

As is, this would break the Chang lab standard which I use for the human recordings.

**Comment by jthermiz** [comment]
@JesseLivezey Sounds good. I updated it to now throw and exception with an explanation to the user if they try to compute wavelets above the Nyquist.

File: process_nwb/wavelet_transform.py [hunk 1]
```diff
@@ -120,6 +121,11 @@ def wavelet_transform(X, rate, filters='rat', hg_only=True, X_fft_h=None, npad=N
         idxs = np.logical_and(cfs >= 70., cfs <= 150.)
         cfs = cfs[idxs]
 
+    # Raise exception if sample rate too small
+    if max(cfs) > rate/2:
+        raise Exception('Unable to compute wavelet transform above Nyquist rate.' +
```
Comment (JesseLivezey) [inline process_nwb/wavelet_transform.py:126]:
```suggestion
        raise ValueError('Unable to compute wavelet transform above Nyquist rate.' +
```

**Review by JesseLivezey — CHANGES_REQUESTED** [review]

**Comment by JesseLivezey** [comment]
I think the failing test is actually my fault.
If you change this to `400.`, I think things should pass
https://github.com/BouchardLab/process_nwb/blob/dd086e4832a37e547588bfbbae2463c8c4becd28/tests/test_wavelet_transform.py#L20

**Comment by JesseLivezey** [comment]
Actually, the fix is a little more complicated. If you fix the flake8 formatting issues, I'll make a follow up PR to fix the test.

**Comment by JesseLivezey** [comment]
If you run `flake8 process_nwb tests` from the root folder, you can see the errors before you push.

**Pull request merged**